sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -186,6 +189,9 @@ def __init__(
       device: the device on which you wish to run the model. If
         ``device = GPU`` then a GPU device will be used if it is available.
         Otherwise, it will be CPU.
+
+    Supported Versions:
+      PyTorch 1.9 and 1.10 are currently supported for Python 3.7, 3.8, and 3.9
```

Comment:
```suggestion
      PyTorch 1.9 and 1.10 are currently supported for Apache Beam.
```

Apache Beam supports those three versions. If a newer Python version gets added, we don't need to update here.

Reply:
Do we know that the Python versions supported by Beam and for each version of PyTorch will always be the same? And would someone reading this necessarily understand that saying these PyTorch versions are supported for Beam is making a claim about with versions of Python it's supported for?

Reply:
The Python versions supported by Beam will change over time, albeit slowly. We're working on 3.10 support right now. 

It should be implied that the PyTorch versions we list here should work for all Beam Python versions we support.

Reply:
Okay, I'll make the change.

Reply:
Change has been added to the PR.

Reply:
Thanks. Instead of "currently supported for", I think using "have been tested in" works better. (It implies that other PyTorch versions could work, we just haven't explicitly tested them)

Reply:
I thought "supported" just meant that we're guaranteeing compatibility for those versions. I don't think either construction necessarily implies that other versions definitely don't work. If anything, "have been tested in" seems to me to be hedging on whether or not people can count on compatibility.

Reply:
I see where you're coming from. Perhaps we could get someone else's take on the wording here. @tvalentyn What do you think?

Reply:
RunInference APIs in Apache Beam have been tested with PyTorch 1.9 and 1.10.